style/popup.css

```diff
@@ -0,0 +1,270 @@
+}
+
+.popup-main-container {
+  width: 100%;
+  height: auto;
+  justify-content: center;
+  display: flex;
+}
+
+.card-popup-window.open-pop-up {
+  display: block;
+}
+
+.popup-main-container .main-container {
+  width: 80%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 2rem;
+  position: relative;
+  z-index: 9999;
+  margin: 3rem 0;
+}
+
+.card-popup-window::-webkit-scrollbar {
+  width: 12px;
+}
+
+.card-popup-window::-webkit-scrollbar-thumb {
+  background: linear-gradient(transparent, #3e4141);
+  border-radius: 6px;
+}
+
+.card-popup-window::-webkit-scrollbar-thumb:hover {
+  background: linear-gradient(transparent, #8a9dff);
+}
+
+.card-popup-window .main-container .title {
+  margin-bottom: 0.5rem;
+}
+
+.card-popup-window .main-container .title h2 {
+  font-size: 28px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.card-popup-window .main-container .info {
+  margin-bottom: 2rem;
+  height: auto;
+  color: #7a869a;
+}
+
+.card-popup-window .main-container .info span {
+  margin-right: 1rem;
+}
+
+.card-popup-window .main-container .info span:first-child {
+  color: #000;
+  font-weight: 600;
+}
+
+.card-popup-window .main-container .card-image {
+  width: 100%;
+  text-align: center;
+}
+
+.card-popup-window .main-container .card-image img {
+  max-width: 100%;
+  width: 100%;
+}
+
+.card-popup-window .main-container .card-image img:hover {
+  transform: scale(1);
+}
+
+.left-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 20px;
+}
+
+.card-popup-window .left-container .text-container {
+  margin-top: 5px;
+  text-align: start;
+}
+
+.card-popup-window .text-container p {
+  font-size: 15px;
+  font-weight: 400;
+  color: #344563;
+}
+
+.card-popup-window .main-container .languages {
+  display: flex;
+  margin-top: 5px;
+
+  /* width: 70%; */
+  flex-wrap: wrap;
+}
+
+.card-popup-window .main-container .languages li {
+  display: none;
+}
+
+.card-popup-window .main-container .languages li:nth-child(1),
+.card-popup-window .main-container .languages li:nth-child(2),
+.card-popup-window .main-container .languages li:nth-child(3) {
+  padding: 0.12rem 0.75rem;
+  background-color: #ebebff;
+  border-radius: 8px;
+  margin: 0.4rem;
+  display: block;
+}
+
+.card-popup-window .main-container .languages li a {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+}
+
+.card-popup-window .main-container .links {
+  width: 100%;
+  margin-top: 2rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-top: 1px solid #ebecf0;
+  padding-top: 2rem;
+}
+
+.card-popup-window .main-container .links .btn {
+  display: flex;
+  align-items: center;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+.card-popup-window .main-container .links .btn:first-child {
+  margin-right: 5px;
+}
```

Comment:
- [ ] Kindly make sure the mobile popup button matches that of the Figma template

Your design
![Capture](https://user-images.githubusercontent.com/85095702/162264915-a228cfbc-2071-4dcb-8970-a558202f9649.PNG)

Figma template
![figma2](https://user-images.githubusercontent.com/85095702/162264996-77596480-249d-4857-a92f-77f9a38e0274.PNG)